apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -165,6 +165,14 @@ function load_editor_plans_grid() {
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_editor_plans_grid' );
 
+/**
+ * Sigh: load_editor_site_launch
```

Comment:
I see this sigh in each item but what does it mean? :-D

Reply:
_Sigh_ is satisfying the linter enforcing these docblocks `load_editor_site_launch` -> `Load editor site launch` 🤷‍♂️ 

There was no need for the _sigh_ commentary in the first place and we probably shouldn't be reproducing it 😅